src/triedb.rs

```diff
@@ -147,6 +145,67 @@ impl<'db> Trie for TrieDB<'db> {
     }
 }
 
+impl<'db> CryptoStructure for TrieDB<'db> {
+    /// A proof creation logic for TrieDB.
+    /// A proof is basically a list of serialized trie nodes, Vec<Bytes>.
+    /// It starts from the one closest to the root, and to the leaf. (It may not reach the leaf in absence case)
```

Comment:
It starts from the one closest to the root and to the leaf. (It may not reach the leaf in absence case.)